node/src/components/chainspec_loader/chainspec.rs

```diff
@@ -301,7 +299,7 @@ impl GenesisConfig {
             timestamp,
             validator_slots,
             auction_delay,
-            locked_funds_period,
+            locked_funds_period_millis: locked_funds_period,
```

Comment:
I'd also rename the local variable then.

Reply:
Whoops, missed that one.

Reply:
Donezo.